docs/components/fieldset/demo/base-demo.md

```diff
@@ -0,0 +1,16 @@
+```hbs template
+<Form::Fieldset
+  @label='Label'
+  @hint='Extra information about the fieldset'
+  @error='Error message'
+>
+  <p class='text-body-and-labels text-xs m-0 italic'>~Fieldset components render
+    here!~</p>
+</Form::Fieldset>
```

Comment:
Essentially how some of this will look is:

```hbs
<Form::Fieldset
  @label='Label'
  @hint='Extra information about the fieldset'
  @error='Error message'
>
    <Form::CheckboxField @label='Label1' />
    <Form::CheckboxField @label='Label2' />
    <Form::CheckboxField @label='Label3' />
</Form::Fieldset>
```

Reply:
A demo for checkboxes and radios (maybe as a follow up) would be great! It seems like the most common use case from what you mentioned.

Reply:
wait...do we only get one demo per component? If think so...if so yeah ^^ I would make those changes.

Reply:
in the default docfy setup (which this is closer to, as opposed to the CS-internal docfy usage), additional demos are appended on the page vertically -- example here: https://docfy.dev/docs/ember/components/docfy-previous-and-next-page
folder for the demos here: https://github.com/josemarluedke/docfy/tree/main/packages/ember/docs/components/docfy-previous-and-next-page-demo